.env.development

```diff
@@ -3,3 +3,4 @@ VITE_DOMAIN_URL=https://testnet-mining-app.hydradx.io
 VITE_INDEXER_URL=https://basilisk-devnet-explorer.play.hydration.cloud/graphql
 VITE_FF_WALLET_ENABLED=true
 VITE_USD_PEGGED_ASSET_NAME=DAI
+VITE_OMNIPOOL_ADDRESS=7L53bUTBbfuj14UpdCNPwmgzzHSsrsTWBHX5pys32mVWM3C1
```

Comment:
Možná nedává smysl to dát jako env ale jako konstantu? Ta adresa by měla by být vždy stejná. (`modlomnipool` s pad right na 32 znaků)